_posts/2019-04-21-dual-boot-with-rails-6-0-beta.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title:  "Getting Ready for Rails 6.0: How to Dual Boot"
+date: 2019-04-21 10:00:00
+categories: ["upgrade-rails", "dual-boot"]
+author: "etagwerker"
+---
+
+[RailsConf 2019](https://railsconf.com) is right around the corner. That means
+[Rails 6.0](https://edgeguides.rubyonrails.org/6_0_release_notes.html) is right
+around the corner! [Rails 6.0's beta](https://weblog.rubyonrails.org/2019/1/22/this-week-in-rails-rails-6-0-0-beta1-and-more/) has been available since January 18, 2019. 🎉
+
+In this article I will explain how you can _dual boot_ your application in your
+local environment and your CI server. I hope that this will help you get ready
+for the next release of Rails.
+
+<!--more-->
+
+Even though my example assumes you are running Rails 5.2 and want to migrate to
+Rails 6.0, these tips work for any two versions of Rails.
+
+## Create a Gemfile.next File
+
+At RailsConf 2018, Jordan Raine talked about Clio's process to upgrade
+Rails over the years. If you missed his talk, you can watch it over here:
+[Ten Years of Rails Upgrades](https://www.youtube.com/watch?v=6aCfc0DkSFo)
+
+In his talk he mentioned quite a handy companion gem: [`ten_years_rails`](https://rubygems.org/gems/ten_years_rails). I'm going to use that gem to get my project ready for dual
+booting. First, I need to install it in my local environment.
+
+```
+$ gem install ten_years_rails
+Successfully installed ten_years_rails-0.2.0
+Parsing documentation for ten_years_rails-0.2.0
+Installing ri documentation for ten_years_rails-0.2.0
+Done installing documentation for ten_years_rails after 0 seconds
+1 gem installed
+```
+
+_Warning: `ten_years_rails` requires Ruby 2.3 or higher. You can find a manual
+workaround below._
```

Comment:
@etagwerker Where's the workaround that you mentioned?

Reply:
@lubc The workaround is here: https://github.com/fastruby/blog/pull/19/files#diff-03f5515211780b3ec09ec0bfe21cb1d6R75 -- would you change anything else to make it clearer?

Reply:
Ah ok, maybe you can add link when you say `below` that scrolls down a bit. But I guess is ok if you keep it like that.